src/poolMetas.ts

```diff
@@ -9,7 +9,10 @@ export class PoolMeta {
   pile: string // pile contract address
   nftFeed: string // NFT or NAV feed contract address
   assessor: string // assessor contract address
+  juniorTranche: string
   seniorTranche: string // senior tranche contract address
+  drop: string
```

Comment:
This is a minor point, but maybe it would make sense to call these `seniorToken` and `juniorToken`, rather than `drop` and `tin`? This aligns better with the other names in this object, as well as the names we're using in the list of contract addresses.